Makefile

```diff
@@ -1,12 +1,11 @@
-.PHONY: all binary build-container build-local clean install install-binary shell test-integration
+.PHONY: all binary build-container build-local clean install install-binary install-completions shell test-integration
 
 export GO15VENDOREXPERIMENT=1
 
 PREFIX ?= ${DESTDIR}/usr
 INSTALLDIR=${PREFIX}/bin
 MANINSTALLDIR=${PREFIX}/share/man
-# TODO(runcom)
-#BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions
+BASHCOMPLETIONDIR=${DESTDIR}/etc/bash_completion.d
 GO_MD2MAN ?= /usr/bin/go-md2man
```

Comment:
This is ont the correct directory.  They should go in 

BASHINSTALLDIR=${PREFIX}/share/bash-completion/completions

/etc is for the admin, not for software to be installed.